nakama/session.lua

```diff
@@ -22,28 +22,67 @@ function M.expired(session)
 	return os.time() > session.expires
 end
 
+--- Decode JWT token
+-- @param token base 64 encoded JWT token
+-- @return decoded token table
+local function decode_token(token)
+	local p1, p2, p3 = token:match(JWT_TOKEN)
+	assert(p1 and p2 and p3, "jwt is not valid")
+	return json.decode(b64.decode(p2))
+end
 
 --- Create a session object with the given data and included token.
--- @param data A data table containing a "token" attribute.
+-- @param data A data table containing a "token", "refresh_token" and other additional information.
 -- @return The session object.
 function M.create(data)
-	local token = data.token
-	assert(token, "You must provide a token")
+	assert(data.token, "You must provide a token")
+	assert(data.refresh_token, "You must provide a refresh token")
```

Comment:
We should not require a refresh token because even after upgrading the SDK the user could have an old-format session with just a regular token. We'll still want them to be able to use that token.

Reply:
Fixed!